### ISSUE TITLE
WCAG Level A: Page Titled

### ISSUE THREAD

**Issue by markhobson** [acteng/update-your-capital-schemes#132]
# High Priority WCAG Level A

## Page Titled

Web pages did not have titles that described topic or purpose.

### WCAG Reference:

2.4.2 Page Titled (Level A)
[Understanding Page Titled](https://www.w3.org/WAI/WCAG22/Understanding/page-titled.html) | [How to Meet Page Titled](https://www.w3.org/WAI/WCAG22/quickref/#page-titled)

## Issue ID: DAC_Page_Titled_01

URL: https://test.update-your-capital-schemes.activetravelengland.gov.uk/schemes
Page title: 'Update your capital schemes - Active Travel England - GOV.UK'
H1 heading: 'Your schemes'
Journey: 1.1a

Screenshot:
![page-titled](https://github.com/user-attachments/assets/e1591535-4420-4a4b-8e68-2478b6c1d014)

On the 'Your schemes' page and subsequent pages in the service the page title remains the same throughout, failing to provide a unique and descriptive title for the page.

Whilst all users can benefit from unique and descriptive page titles, this particularly affects screen reader users as the page title is the first thing that screen readers announce on page load and is often the primary means for screen reader users to identify content and distinguish different pages to orientate themselves within the service.

### Current code ref(s):

head > title
```html
<title>
   Update your capital schemes - Active Travel England - GOV.UK
</title>
```

### Screen reader user comments:

> “When I activated the reference link there was no change in the page title.
> This is an issue as a screen reader user would be unaware that the page had changed, and they would remain unaware until they found the h1 which within its heading text gives full information of the page’s content. I feel that ensuring page titles change to reflect the content would eliminate any issues that at present may be encountered by a screen reader user navigating this area of the service.”

### Examples of additional instances:

* URL: https://test.update-your-capital-schemes.activetravelengland.gov.uk/schemes/64
  Page title: 'Update your capital schemes - Active Travel England - GOV.UK'
  H1 heading: 'BO-ATF-001 Manchester Road Corridor Phase 1' (scheme overview page)
  Journey: 1.2a

* URL: https://test.update-your-capital-schemes.activetravelengland.gov.uk/schemes/64/spend-to-date
  Page title: 'Error: Update your capital schemes - Active Travel England - GOV.UK'
  H1 heading: 'Change spend to date'
  Journey: 1.3b

* URL: https://test.update-your-capital-schemes.activetravelengland.gov.uk/schemes/64/milestones
  Page title: 'Error: Update your capital schemes - Active Travel England - GOV.UK'
  H1 heading: 'Change milestone dates'
  Journey: 1.5a

Additional instances of this issue may exist on other pages throughout the website; wherever this issue occurs, they too will need to be resolved.

### Solution:

Ensure that page titles are descriptive of the topic or purpose of the page, and are unique within the service.

As detailed in [GOV.UK Service Manual: Writing for user interfaces (Headings and <title>)](https://www.gov.uk/service-manual/design/writing-for-user-interfaces#headings-and-title), but which is explained in better detail in [HMRC Design System: Page Title](https://design.tax.service.gov.uk/hmrc-design-patterns/page-title/), page titles should either follow the format:
```
H1 heading – service name – GOV.UK
```
Or, if content has been further divided into sections, it should also include the section name:
```
H1 heading – section name – service name – GOV.UK
```
Where the page title is 'front loaded' to place the most specific and relevant information first, to permit screen reader users to skip the rest of the page title announcement, once they have heard enough to orientate themselves within the service.

### Example:
```html
<title>
   Your schemes - Active Travel England - GOV.UK
</title>
```
